### ISSUE TITLE
datetime read from google spreadsheet gives inconsistent results

### ISSUE THREAD

**Issue by shuodeng** [google/google-visualization-issues#2278]
In my JavaScript, I pulled a series of data points from a google spreadsheet and plot charts using https://www.google.com/jsapi

It was working fine a few days ago. But today, I noticed the api is reading "2016-2-27" to be "Sun Mar 27 2016", while reading "2016-1-16" to be "Sat Jan 16 2016". Why is one date shifted by one month but the other one isn't?

**Comment by lexbrugman** [comment]
I'm seeing similar issues with the 'annotationchart', some datapoints are off by one month. In my case the data source is not a google spreadsheet but a JSONP request.

**Comment by dtomczyk1978** [comment]
Seeing the same exact thing with a LineChart. It was working fine last week and it just broke yesterday. Using a JSON request.

**Comment by seahrh** [comment]
I have reported the same issue [on the forums](https://groups.google.com/d/msg/google-visualization-api/u-QPyDTmsyc/7s8hXOmZBQAJ) and have at least 1 other having the same problem. 

Link chart draws funny kinks when plotting February data points in dd/mm/yyyy format.

e.g. see my first two charts: http://coe.sgcharts.com/

The dates were
03/02/2016 - drawn incorrectly as March
17/02/2016 - drawn incorrectly as March
09/03/2016 - OK
23/03/2016 - OK

and affected all Feb data points in previous years.

Backing spreadsheet:
https://docs.google.com/spreadsheets/d/1Ma8dm_rdtdfNp8ONUG5ykFHwrEg1GFC3ObOMualMVBM/edit?usp=sharing

The charts look fine 2 weeks ago and I have not touched the code for a long while. Was wondering what happened.

**Comment by iFrancs** [comment]
I got the same issue, my calendar chart is showing the information in a bad way

Here's an example of the bad interpretation of the datetime
http://jsfiddle.net/asewLmdz/2/

**Comment by travisbell** [comment]
Yup, same here. All of my charts are showing super random dates when they're all just using chronological date values. Everything was working fine until about 2 weeks go.

**Comment by dlaliberte** [comment]
This is a new bug involving parsing dates when specified with date-strings.   We have a fix, but it will likely be a couple weeks before we release it. If you can avoid using date-strings, you should have better luck in the meantime.

Another workaround is to switch to using the new loader:  https://developers.google.com/chart/interactive/docs/basic_load_libs#update-library-loader-code

**Comment by dtomczyk1978** [comment]
It's not just date-strings. I'm using the Date constructor in my JSON and
I'm having the same problems. Eg. Date(2016,0,29).

**Comment by chrisbeaver** [comment]
This is working for me as of now, I was experiencing this issue yesterday.

**Comment by dlaliberte** [comment]
It is only working now because it is no longer the end of the month.  It will start failing again on June 29th. So we will have to push out a real fix before then.
